idd/Energy+.idd.in

```diff
@@ -74792,6 +74792,8 @@ SetpointManager:OutdoorAirReset,
   A2 , \field Control Variable
        \type choice
        \key Temperature
+       \key MaximumTemperature
+       \key MinimumTemperature
```

Comment:
The IDD memo for SetpointManager:OutdoorAirReset, and the I/O Ref description use the term "supply air" temperature which is misleading since it can also be used for plant loops.  These need to be edited to speak simply of a setpoint temperature.  "This setpoint manager is used to place a temperature setpoint on a system node . . . "